README.md

```diff
@@ -1,24 +1,11 @@
 # hyper-content-for
 
-Allows you to store HTML for use (and re-use) later.
```

Comment:
I would like to keep a small summary underneath the top heading, as this seems to be a common practice and I like to be able to quickly just read a short version of the entire description just to get a feel of what the thing does. Sort of like Git commit messages, I suppose; you'd have the short summary of what is included in the commit and then maybe a longer description underneath that again.

I agree that the current summary (the one you removed) is kinda lame, though, so I'd very much like to swap it out with something more meaningful. :smiley: